src/server/db/resources.js

```diff
@@ -0,0 +1,637 @@
+const topics = [
+  'Before you Start a Social Business',
+  'Starting a Social Business',
+  'Running a Social Business',
+  'Growing A Social Business',
+  'Case Studies',
+];
+
+const classes = [
```

Comment:
we should dispose entirely with these json files and hold the data in insert statements. you can generate that script programatically of course, but we don't want these included in the pr.